client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -646,15 +646,17 @@ const Editor = (props: EditorProps): JSX.Element => {
     }
   };
 
-  function initializeProjectFeatures() {
+  function initializeProjectStepFeatures() {
     const editor = data.editor;
     if (editor) {
       initializeRegions(getEditableRegionFromRedux());
       addWidgetsToRegions(editor);
     }
   }
 
-  function isProject() {
+  // Currently, only practice project parts have editable region markers
+  // This functions is used to enable multiple editor tabs, jaws, etc.
```

Comment:
```suggestion
  // These functions is used to enable multiple editor tabs, jaws, etc.
```

Reply:
"This function" refers to `isProject`, so there's only one.

Reply:
got it.

Reply:
I updated my requested change accordingly